src/components/BottomSheet/BottomSheet.js

```diff
@@ -13,34 +14,31 @@ const BottomSheet = React.forwardRef((props, ref) => {
       onOpen={() => (props.onOpen ? props.onOpen() : {})}
       closeOnDragDown={true}
       closeOnPressMask={props.closeOnPressMask}
-      height={props.height ? props.height : 260}
+      height={props.height ? props.height : 355}
       customStyles={{
         container: styles.containerSheet(pullBottom),
-        draggableIcon: styles.draggableIcon,
+        draggableIcon: styles.draggableIcon
       }}>
-      <View style={{...styles.container, ...props.viewstyle}}>
-        {props.children}
-      </View>
+      <View style={{...styles.container, ...props.viewstyle}}>{props.children}</View>
     </RBSheet>
   );
 });
 
-BottomSheet.displayName = 'BottomSheet'
+BottomSheet.displayName = 'BottomSheet';
 
 export default BottomSheet;
 
 const styles = StyleSheet.create({
   container: {
-    paddingHorizontal: 20,
-    paddingTop: 20,
-    paddingBottom: 38,
+    paddingTop: dimen.normalizeDimen(10),
+    paddingHorizontal: dimen.normalizeDimen(20)
   },
   containerSheet: (pullBottom) => ({
     borderTopRightRadius: 20,
     borderTopLeftRadius: 20,
-    justifyContent: pullBottom ? 'flex-end' : 'flex-start',
+    justifyContent: pullBottom ? 'flex-end' : 'flex-start'
   }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The paddingBottom style has been removed from the container style and justifyContent has been moved to the containerSheet style. This could potentially alter the layout and alignment of elements within the BottomSheet. Please ensure that these changes are intentional and tested thoroughly.
```diff
-    paddingBottom: 38,
-    justifyContent: pullBottom ? 'flex-end' : 'flex-start',
+    containerSheet: (pullBottom) => ({
+      borderTopRightRadius: 20,
+      borderTopLeftRadius: 20,
+      justifyContent: pullBottom ? 'flex-end' : 'flex-start'
+    }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->